todo_app/trello_api.py

```diff
@@ -0,0 +1,82 @@
+import os
+import requests
+
+class TrelloApi:
+
+  base_url = os.getenv('TRELLO_API_BASE_URL')
+  board_id = os.getenv('TRELLO_BOARD_ID')
+  token = os.getenv('TRELLO_API_TOKEN')
+  key = os.getenv('TRELLO_API_KEY')
+
+  @classmethod
+  def get_cards(cls):
+    url = f"{cls.base_url}/1/boards/{cls.board_id}/lists"
+
+    params = {
+        "key": cls.key,
+        "token": cls.token,
+        "cards": "open",
+        "card_fields": "id,idList,name"
+      }
+    
+    try:    
+      response = requests.get(url, params=params)
+      return response.json()
+    except Exception as e:
+      print(f"An error occurred: {e}")
+      return None
```

Comment:
I'm always a bit wary of this `return None` pattern - if you're thinking about exceptions this has meant:
* We've reported an error in the logs
* But _probably_ some later bit of code is just going to throw an unhelpful `NoneType` exception because of this

So the user still sees a generic error, and our stack trace is less clear. I'd argue for either:
* Let the error actually be raised, or print some helpful logging then raise it again, if we're happy to trigger later error handling
* Or record that the error has happened here, and return something of the right structure, e.g. an empty list here to imply no cards were seen if we want to "hide" the error from the user
  * (For example, if you were returning a subset of the information on a page, maybe we want the rest of the page to load healthily and we just want one section to be blank because of the error in that area)

Reply:
Ah, that makes sense, thanks! Where the attempt to get cards is made I've opted to `return []` in the except block, and for the other methods in this file still print the exception message but then `raise` instead of `return None`. I'll need to add proper error handling in `trello_items.py` to accommodate this at some point!